introduction.md

```diff
@@ -4,3 +4,4 @@ I will probably like:
 *Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?